content/metadata.yaml

```diff
@@ -26,14 +26,75 @@ authors:
       - Investigation
       - Methodology
       - Validation
+      - Visualization
       - Writing - Original Draft
+
+  - github: janeroe
+    name: Vince
+    initials: JR
+    orcid: XXXX-XXXX-XXXX-XXXX
+    twitter: null
+    email: jane.roe@whatever.edu
+    affiliations:
+      - Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania
+    contributions:
+      - Resources
+      - Software
+      - Visualization
+      - Writing - Review and Editing
+
+
+  - github: janeroe
+    name: Dongbo
+    initials: JR
+    orcid: XXXX-XXXX-XXXX-XXXX
+    twitter: null
+    email: jane.roe@whatever.edu
+    affiliations:
+      - Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania
+    contributions:
+      - Resources
+      - Software
       - Visualization
+      - Writing - Review and Editing
+
+  - github: janeroe
+    name: Marvin
+    initials: JR
+    orcid: XXXX-XXXX-XXXX-XXXX
+    twitter: null
+    email: jane.roe@whatever.edu
+    affiliations:
+      - Elsevier stub here
+    contributions:
+      - Conceptualization
+      - Data Curation
+      - Formal Analysis
+      - Methodology
+      - Writing - Review and Editing
 
   - github: janeroe
-    name: Jane Roe
+    name: Larry
```

Comment:
Can you fill in this author info from folks prior publications? At least github, full names, initials, and ORCIDs if you can find them